src/PhpAssumptions/Analyser.php

```diff
@@ -35,11 +35,11 @@ class Analyser
     private $result;
 
     /**
-     * @param ParserAbstract $parser
+     * @param ParserAbstract         $parser
      * @param NodeTraverserInterface $nodeTraverser
      */
     public function __construct(
-        ParserAbstract $parser,
+        Multiple $parser,
```

Comment:
I don't think that's quite right. From what I'm seeing Cli.php creates the class being passed into this construct here:

```
    private function createParser()
    {
        $parser = (new ParserFactory)->create(ParserFactory::PREFER_PHP7);
        return $parser;
    }
```

Since this results in the creation of an instance of: PhpParser\Parser\Multiple I'm type hinting for that class.